sktime/transformations/panel/rocket/_minirocket_multivariate_variable.py

```diff
@@ -0,0 +1,1185 @@
+# -*- coding: utf-8 -*-
+"""Multivariate MiniRocket transformer."""
+
+__author__ = ["angus924", "michaelfeil"]
+__all__ = ["MiniRocketMultivariateVariable"]
+
+import multiprocessing
+import warnings
+from typing import List, Union
+
+import numpy as np
+import pandas as pd
+from numba import get_num_threads, njit, prange, set_num_threads, vectorize
+
+from sktime.transformations.base import BaseTransformer
+
+
+class MiniRocketMultivariateVariable(BaseTransformer):
+    """MINIROCKET (Multivariate, unequal length).
+
+    MINImally RandOm Convolutional KErnel Transform. [1]_
+
+    **Multivariate**
+    **unequal length**
+
+    A provisional and naive extension of MINIROCKET to multivariate input
+    with unequal length provided by the authors [2]_ .  For better
+    performance, use the sktime class MiniRocket for univariate input,
+    and MiniRocketMultivariate to equal length multivariate input.
+
+    Parameters
+    ----------
+    num_kernels              : int, number of random convolutional kernels
+                               (default 10_000).
+                               Reduced to the closest multiple of 84.
```

Comment:
[This](https://sktime--3786.org.readthedocs.build/en/3786/api_reference/auto_generated/sktime.transformations.panel.rocket.MiniRocketMultivariateVariable.html#sktime.transformations.panel.rocket.MiniRocketMultivariateVariable) is the generated docs for the estimator. There are a couple of issues, but the other transformers also have them. Not a requirement to fix, but making you aware.

Reply:
Okay, should be possible to fit!